server/src/main/java/org/opensearch/search/backpressure/CancellationListener.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.backpressure;
+
+/**
+ * Listener for callbacks related to cancellation settings
+ */
+public interface CancellationListener {
```

Comment:
nitpick: `CancellationListener` sounds like something related to task cancellations. But these callbacks are for token bucket settings.

Reply:
Okay, will rename it to `SearchBackpressureCancellationSettingsListener`.